src/components/pager/pager.stories.js

```diff
@@ -0,0 +1,56 @@
+import { html } from "lit-html";
+import "../pager";
+import { docsIntro } from "../../../.storybook/utils.js";
+import { args, argTypes } from "./config";
+
+export default {
+  title: "custom-elements/vl-pager",
+  args,
+  argTypes,
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          root: "pager",
+          intro:
+            "Use the pager component to show the number of available pages, highlight the current page (range) and add previous/next buttons for navigation.",
+        }),
+      },
+    },
+  },
+};
+
+export const Default = ({
+  totalItems,
+  itemsPerPage,
+  currentPage,
+  paginationDisabled,
+  alignCenter,
+  alignRight,
+}) => html`
+  <vl-pager
+    data-vl-total-items=${totalItems}
+    data-vl-items-per-page=${itemsPerPage}
+    data-vl-current-page=${currentPage}
+    ?data-vl-pagination-disabled=${paginationDisabled}
+    ?data-vl-align-center=${alignCenter}
+    ?data-vl-align-right=${alignRight}
+  ></vl-pager>
+`;
+
+export const SinglePage = () => html`
```

Comment:
Als je cases van bepaalde combinatie van attributen wilt tonen in een story, zet ze er dan niet hardcoded in maar gebruikt templates, met bepaalde attributen getoggled vanuit uw story. Zoals bv hier; https://uig-webcomponents.omgeving.vlaanderen.be/?path=/story/custom-elements-vl-infoblock--contact

Reply:
FIxed.